docs/use-mainnet/bridges-of-linea/index.mdx

```diff
@@ -7,17 +7,40 @@ sidebar_position: 3
 
 Before you begin, make sure you've [set up your wallet](/use-mainnet/set-up-your-wallet)
 
+
+
+### For all normal bridge traffic: 
+
+#### **We recommend you use the Bridge feature of MetaMask Portfolio: [https://portfolio.metamask.io/bridge](https://portfolio.metamask.io/bridge)**
+>
+>The Linea ecosystem has a **growing and diverse number of bridges** operating on it. 
+>
+>MetaMask Portfolio's Bridge feature **aggregates all those bridges**, and shows the options available to you so that you can **choose the provider and rate that you prefer**. Then, you can perform the bridging operation right from the dapp.
```

Comment:
Bridge only uses a select handful of providers, which, in turn, are accessed through just two aggregators: Li.Fi and Socket. Suggest we amend to not say that Bridge uses _all_ bridges, and instead make it slightly more vague. 

see "What aggregators and providers do you use" in the FAQs [here](https://support.metamask.io/hc/en-us/articles/10055915089819-How-to-use-the-bridge#h_01GGSRF9FCZR53YCNSME5TMMY2).

Reply:
Thank you for actually knowing the technical details....! This was me, assuming it was like Swaps.